ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -426,25 +426,25 @@ def _router_bgp_vpws(self) -> list[dict] | None:
 
                     # Endpoints can only have two entries with index 0 and 1.
                     # So the remote must be the other index.
-                    remote_index = int(not (local_index))
+                    remote_endpoint = endpoints[int(not (local_index))]
```

Comment:
so for me this looks ok but it is a bit magic with type changes.
This works as well and at least shows we are manipulating int and not `int(bool(int)))`

```suggestion
                    remote_endpoint = endpoints[(local_index + 1) % 2]
```

I timed it on it is roughly the same so .. your choice! :)

Reply:
fixed